marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,4 @@
+Header header
+
+bool   healthy
+string message
```

Comment:
That seems reasonable as long as we document what the different levels mean to each node that's using it.